ts/Extensions/MouseWheelZoom/MouseWheelZoom.ts

```diff
@@ -66,198 +70,257 @@ const optionsToObject = (
 };
 
 /**
+ * Fit the new extremes inside axis bounds.
  * @private
+ * @param {Highcharts.Chart} outerStart
+ * Beggining of bounds range.
+ * @param {Highcharts.Axis} outerWidth
+ * Width of bounds range.
+ * @param {Highcharts.Chart} innerStart
+ * Beggining of new extremes.
+ * @param {Highcharts.Axis} innerWidth
+ * Width of new extremes.
+ * @return {Object}
+ * Object containing rangeStart and rangeWidth.
  */
```

Comment:
Not true really, because this function does not need any axis knowledge, it just fits a segment inside some range. and the params have the wrong type, those are just numbers.

Reply:
Thanks for the suggestion!